index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <title>Chocolate Pizza</title>
+  </head>
+  <body id='master'>
+    <div id="heading">
+      <div id="topleft">
+        <img id='logo'src="logo.png" alt="logo.png">
+        <span id = 'delicious'>Delicious</span>
```

Comment:
keep the spacing between `id` `=` and the assignment consistent, also when in html everything should be a double quote.